src/sass/App.scss

```diff
@@ -9,6 +9,7 @@ $fa-font-path: "~font-awesome/fonts";
 @import "../components/RichEditor/RichEditor.scss";
 // Required to have the Collapsible component work
 @import '../../node_modules/@edx/paragon/src/Collapsible/Collapsible';
+@import '../components/EditCoursePage/CollapsibleCourseRunFields';
```

Comment:
Nit: our scss files should be grouped together apart from the `node_modules` ones

Reply:
cleaned the file up a bit